README.md

```diff
@@ -75,15 +84,60 @@ class YourApplication : Application() {
 }
 ```
 
-### iOS
-Ideally you will call this in `applicationDidFinishLaunching` in AppDelegate or in `init` in your SwiftUI App
-
+### Cocoa
 ```Swift
 import shared
 
-// ...
+class AppDelegate: NSObject, UIApplicationDelegate {
+    func application(
+        _ application: UIApplication,
+        didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey : Any]? = nil
+    ) -> Bool {
+        Sentry.shared.doInit() { options in
+          options.dsn = "__DSN__"
+        }
+        return true        
+    }
+}
+```
+
+## Shared Initializer
```

Comment:
Yes this is something the user needs to do if they want to use a shared initializer, it's an extra step but if they don't want to do that they can also just initialize directly without doing anything extra. But I think it's also a good thing to document how to do it since code sharing is the main concept in KMP.